src/components/Comments.jsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from "react";
+import { getCommentsByArticleId } from "../utils/utils";
+
+import Comment from "./Comment";
+
+const Comments = ({ article_id }) => {
+  const [comments, setComments] = useState([]);
+
+  useEffect(() => {
+    getCommentsByArticleId(article_id).then((data) => {
+      setComments(data.comments);
+    });
+  }, []);
+
```

Comment:
You could have something to indicate that the comments are loading - the same for the articles list and individual articles.